src/cmdInventoryExport.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type HostVars map[string]interface{}
+type Hosts map[string]HostVars
+
+type Group struct {
+	Hosts []string               `json:"hosts,omitempty"`
+	Vars  map[string]interface{} `json:"vars,omitempty"`
+}
+
+type Groups map[string]Group
+
+type Meta struct {
+	Hostvars Hosts `json:"hostvars"`
+}
+
+type AnsibleInventory struct {
+	Groups Groups `json:"groups"`
+	Meta   Meta   `json:"_meta"`
+}
+
+func (a AnsibleInventory) MarshalJSON() ([]byte, error) {
+	aux := make(map[string]interface{})
+
+	for key, value := range a.Groups {
+		aux[key] = value
+	}
+
+	aux["_meta"] = a.Meta
+
+	return json.Marshal(aux)
+}
+
+type inventoryAnsibleCmd struct{}
```

Comment:
Re previous comment, in that case, if you drop this, the parser may produce "unknown parameter" error. Leave it in, just hidden and so it doesn't do anything:
```
List bool `long:"list" description:"List Ansible Inventory" hidden:"true"`
```

Reply:
nevermind, we do not need this as you override the os.Args in main.go anyways.